internal/action/show.go

```diff
@@ -255,20 +256,23 @@ func (s *Action) showGetContent(ctx context.Context, sec gopass.Secret) (string,
 	}
 
 	// everything but the first line.
-	if ctxutil.IsShowSafeContent(ctx) && !ctxutil.IsForce(ctx) {
-		body := showSafeContent(ctx, sec)
+	if config.Bool(ctx, "core.showsafecontent") && !ctxutil.IsForce(ctx) {
+		debug.Log("core.showsafecontent is true")
```

Comment:
Is this log really meant to be added only for that case and not for all the previous ones?

Reply:
Which previous ones?

Reply:
All the previous `if/else` are not logged, but just this one, so I was wondering it there was a special reason to log this?

Reply:
This was a stray debug statement. We could leave it, but we can also remove it. The other clauses are not new, so they don't have it.